### ISSUE TITLE
[Screen Reader-PowerApps Admin-Tasks ] Buttons must have discernible text for column header and context menu control

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#6693]
### Please provide a reproduction of the bug in a codepen:
 
User Experience:
User depending on screen reader might get affected if buttons will not have discernible text and it will impact the end user .

### Bug Report 
Power platform admin center has an accessibility issue that needs investigation.

Issue Details
Buttons must have discernible text
button-name 

- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
Repro Steps:
1.Navigate to https://admin.powerplatform.microsoft.com/ext/DataIntegration

and login with intusr.
2. Navigate to the Data Integration menu item on the left bar and select it.
3. Navigate on the controls of data integration page thus opened.
4. Navigate to 'Projects' tab and select it . 
5. Navigate to any already existing project and select it. Project edit screen will open.
6. Navigate to 'Tasks' tab and select it.
 
#### Actual behavior:
 
 Buttons do not have discernible text for column header and context menu control.
 
#### Expected behavior:
 Buttons must have discernible text for column header and context menu control.

Application
Power platform admin center
https://admin.powerplatform.microsoft.com/ext/DataIntegration
 
Element Path
#iframeComponent;#header333-columnTaskActions
 
Snippet
<span id="header333-columnTaskActions" aria-labelledby="header333-columnTaskActions-name " class="ms-DetailsHeader-cellTitle cellTitle_d6a41446" data-is-focusable="true" role="button" aria-describedby="header333-columnTaskActions-tooltip" aria-haspopup="fa...
 
How to fix
Fix any of the following:
- Element has a value attribute and the value attribute is empty
- Element has no value attribute or the value attribute is empty
- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"
- Element has no title attribute or the title attribute is empty
 
![mas4 1 2_button 20discernible 20text](https://user-images.githubusercontent.com/30456910/46938872-e9610200-d075-11e8-8ce0-953bf11801a0.jpg)
![overflowset](https://user-images.githubusercontent.com/30456910/46938880-ed8d1f80-d075-11e8-8370-40c9b9cd548f.PNG)

To reproduce
Use K4W to investigate the issue details
 
Environment
Chrome version 69.0.3497.92

**Comment by micahgodbolt** [comment]
Spoke with author today, and they said they were working on codepen repros for this and other related bugs.
